code/modules/client/verbs/ooc.dm

```diff
@@ -100,9 +97,12 @@ GLOBAL_VAR_INIT(normal_ooc_colour, OOC_COLOR)
 	set name = "Set Your OOC Color"
 	set category = "Preferences"
 
+	/*
+	Not sure how analy retarded you made this exactly TG.
```

Comment:
I think it's better to do the fancy patreon nonsense first, instead of unlocking everything, and locking it again later. (I'm referring to stuff like view range and especially character slots) People really really don't like locking existing stuff behind donator. We won't be able to restrict it again. We had that discussion back when goof wanted to lock cats to donators.